src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/LogConfiguration.kt

```diff
@@ -30,11 +30,6 @@ internal class LogConfiguration(
                     null
                 )
             },
-            logToFile(
```

Comment:
Why not have both (events and logs)?

Reply:
Unnecessary tax and dependency. If a consumer wants logs, they can subscribe and log. They don't need it by default thanks to GUI.

Reply:
Also, that behavior change is caused by switching from `TaskTimer` to `TaskScope`, not from this log config. This is just removing dead code here.